_posts/2018-05-28-InfoGAN.markdown

```diff
@@ -25,10 +25,10 @@ stone towards research in representation learning.
 # 1 Information Theory
   **Motivation**: Information theory formalizes the concept of the "amount of randomness" or 
 "amount of information". These concepts can be extended to relative quantities 
-among random variables. This section leads to Mutual Information, a concept core to 
-InfoGAN which extends entropy to how much additional information you yield from 
-observing a joint sample of two random variables as compared to the baseline of 
-observing each variable separately. 
+among random variables. This section leads to Mutual Information (MI), a concept core to 
+InfoGAN. MI extends entropy to the amount of additional information you yield from 
+observing a sample from the conditional distribution of two random variables as compared to the baseline of 
```

Comment:
I don't follow this.

What I was going for here is [this](https://en.wikipedia.org/wiki/Mutual_information#Relation_to_Kullback%E2%80%93Leibler_divergence).